### PR TITLE
UPDATE MENTORS.YML

### PR DESCRIPTION
**Pull request by ShivamRai2003**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — APPROVED** [review]

**Comment by ShivamRai2003** [comment]
PLEASE GET IT MERGE

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@ShivamRai2003 Please add mentor to the last else looks fine 👍

File: _data/mentors.yml [hunk 1]
```diff
@@ -292,3 +292,9 @@
   github: tstreamDOTh
   twitter: artamusebia
   image: Thiyagaraj_T.jpg
+  
+  - name: Sean McGregor
```
Comment (0x48piraj) [inline _data/mentors.yml]: Indent

**Review by 0x48piraj — COMMENTED** [review]
Closed ?

**Pull request closed without merging**